integration-tests/config.ts

```diff
@@ -43,6 +45,9 @@ export const CONFIGS: ConfigWithSetup[] =
       const Tezos = new TezosToolkit();
       if (forger === ForgerType.LOCAL) {
         Tezos.setProvider({ rpc, forger: localForger })
+      } else if (forger === ForgerType.COMPOSITE) {
+        const composite = new CompositeForger([new RpcForger(Tezos['_context']), localForger]);
```

Comment:
We should provide a better of creating instance of the RpcForger as it depends on the `context` which is not exposed to the end user